packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -8,9 +8,7 @@ import 'events.dart';
 
 // Add `kPrimaryButton` to [buttons] when a pointer of certain devices is down.
 //
-// TODO(tongmu): This patch is supposed to be done by embedders. Patching it
-// in framework is a workaround before [PointerEventConverter] is moved to embedders.
-// https://github.com/flutter/flutter/issues/30454
+// Needs Update
```

Comment:
@dkwingsmt The associated issue here is closed, can this be resolved?

Reply:
It can be resolved by https://github.com/flutter/flutter/pull/55805, but it is unfortunately a (minor) breaking change.

Reply:
Awesome! Thanks for opening a PR! I wouldn't consider it high priority, I've just been auditing ToDos, so take your time. Thanks for resolving!